test/tests/table.ts

```diff
@@ -330,11 +361,20 @@ suite('Table', () => {
         })
 
         test('should just get first row without params', async function () {
-            const table = eosio.table<EosioGlobalState>('global', EosioGlobalState)
+            const table = eosio.table<EosioGlobalState>('global', undefined, EosioGlobalState)
             const row = await table.get()
             assert.instanceOf(row, EosioGlobalState)
             assert.instanceOf(row.pervote_bucket, Int64)
         })
+
+        test('should use scope from table call', async function () {
+            const contractKit = new ContractKit({
+                client: makeClient('https://jungle4.greymass.com'),
+            })
+            const contract = await contractKit.load('eosio.token')
+            const result = await contract.table('accounts', 'wharfkittest').get()
+            assert.instanceOf(result.balance, Asset)
```

Comment:
Does this test really make sure that the correct scope is used? 🤔 Wouldn't the balance always be an Asset?